run.go

```diff
@@ -588,60 +627,131 @@ loop:
 	}
 	fmt.Fprintf(color.Output, "\n")
 
+	// Print summary
+	summaryError := printSummary(fs, summaryFile, summaryFormat, engine, atTime)
```

Comment:
Just call this `err`; I think this can be shortened to `if err := printSummary(...); err != nil` without the line length getting ridiculous too?